### ISSUE TITLE
Automatic snapshots for `-Syu` too

### ISSUE THREAD

**Issue by xfzv** [fosskers/aura#879]
With the following in `~/.config/aura/config.toml`:

```toml
[backups]
automatic = true
```

`aura` will automatically create a package set snapshot when running `aura -Au`. This feature is really convenient.

Any chance we could also enable automatic snapshots in the config file for Pacman updates when running `aura -Syu`?

**Comment by fosskers** [comment]
Yes actually, since I entirely control the capturing of arguments to the underlying `pacman`. 

One critical difference between `-Su` and `-Au` would be that with `-Au`, I wait until after the user has accepted the upgrade before I save the package state. With `-Su` I'd have no way of doing that, since by that point control has already been relinquished to `pacman`. If that sounds reasonable, I could implement this.
